docs/inputs/input-group.md

```diff
@@ -48,14 +52,14 @@ As an example, the image below shows an input group with 5 different inputs (two
 | :trophy: This is a [premium](https://mecsimcalc.com/pricing) feature |
 | -------------------------------------------------------------------- |
 
-Click the :pencil2: icon near the top right of an input group and the **Edit Input Group** popup will appear.  
-**Conditional Input Group** is similar to the [Conditional Input](/inputs/conditional-input) feature, but instead effects the entire **Input Group**.
+Click the ![img alt](/docs/getting-started/edit_btn.png) icon near the top right of an **Input Group** and the **Edit Input Group** popup will appear.  
+**Conditional Input Group** is similar to the **[Conditional Input](/inputs/conditional-input)** feature, but instead effects the entire **Input Group**.
```

Comment:
"affects" instead of "effects", should be "but instead affects the entire **Input Group**."